themes/default/content/blog/stack-init-teams-flag/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Announcing the Stack Init Teams Flag"
+date: "2023-03-28"
+meta_desc: "We've added a new CLI flag to the stack init subcommand allowing users to grant team access to newly created stacks."
+meta_image: "meta.png"
+
+authors:
+    - "robbie-mckinstry"
+        
+tags:
+    - "cli"
+    - "stack-init"
+    - "cli-flags"
+---
+
+We recently released a quality-of-life feature for the Pulumi CLI to save users time and energy when creating stacks. Users can now initialize team permissions when creating stacks on the command line.
+
+<!--more-->
+
+For context, here's some background for any Pulumi newcomers. Pulumi projects
+consist of stacks—separate deployments of the same infrastructure. Stacks may
+also be configured with separate inputs. Organization admins can leverage RBAC
+to  grant project and stack access by sorting members into roles called "teams."
```

Comment:
```suggestion
to grant project and stack access by sorting members into roles called "teams."
```

Reply:
is this really how we tell users to think about this? that teams are roles? 🤔

Reply:
@susanev that's how _I_ think about this! Maybe marketing and product feel differently!
But when I think of Pulumi Teams, I think of RBAC. It looks like that's how we pitch it, too:
https://www.pulumi.com/docs/intro/pulumi-service/teams/
> # Teams and RBAC
> The Pulumi Service offers role-based access control (RBAC) using teams. Teams allow organization admins to assign a set of stack permissions to a group of users.